packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -324,6 +324,7 @@ class ScrollableBase extends Component {
 	lastScrollPositionOnFocus = null
 	indexToFocus = null
 	nodeToFocus = null
+	pointToFocus = null
```

Comment:
I haven't had time to review this ticket in detail yet but wanted to ask about this specifically. Can you share why you chose to use a non-state member here rather than using `setState`?

Reply:
@ryanjduffy 
Related methods to store and restore that info are invoked by key input and already out of life cycle. If I make it a _state_, I have to check it is safe to _scroll_ when `setState` is invoked by key press. So, simply I made it a variable which is managed by imperative calls.